### PR TITLE
Tracker Touchups

### PR DESCRIPTION
**Pull request by gymnast86**
## What does this address?
- Adds a Notes section to the bottom right of the tracker that users can write in and will be autosaved along with everything else
- Changes gratitude crystals to count by 1 instead of 5. 
- Yeets the separate images that were used for items with number counters and draws the numbers over the image as a label instead

## How did/do you test these changes?
I clicked through all the gratitude crystal numbers and it looked good. I then wrote down some notes and they were properly saved after closing and reopening the program. I tested different fonts for the tracker number labels and they all worked well enough for me to be satisfied.

### CODE REVIEW

**Comment by CovenEsme** [comment]
Looks good. Only thing I noticed was that clicking "Start New Tracker" didn't reset the notes section

**Comment by gymnast86** [comment]
That was an intentional decision. Idk why but it felt more natural to me to let the user delete their own notes instead of having them reset when clicking on start new tracker. If it doesn't feel natural to you though, then we can delete them on starting a new tracker.

**Comment by CovenEsme** [comment]
Yeah, that's fair. It's just something I noticed - I'm happy to leave it how it currently behaves :p

Might be worth adding a "Clear Notes" button? I guess it depends how cluttered the tracker UI is getting

**Comment by gymnast86** [comment]
Probably not necessary. You can just ctrl + A or highlight and then delete all your notes which is fast enough imo

**Review by CovenEsme — APPROVED** [review]

File: gui/tracker.py [hunk 1]
```diff
@@ -1659,6 +1669,25 @@ def on_click_inventory_button(self, item: Item, item_image: str):
...
+                    "1 crystal",
+                    "5 crystals",
+                )
+                if single_crystal_question != QMessageBox.StandardButton.Yes:
+                    item = self.world.get_item(GRATITUDE_CRYSTAL_PACK)
+                    crystals_button = self.gratitude_crystals_button
+                    for _ in range(4):
+                        crystals_button.state += 1
+                        if crystals_button.state >= len(crystals_button.items):
+                            crystals_button.state = 0
```
Comment (YourAverageLink) [inline gui/tracker.py:1687]: We need to make sure the extra crystals are added to the inventory too.
```suggestion
                    for _ in range(4):
                        crystals_button.state += 1
                        crystals_button.add_current_item()
                        if crystals_button.state >= len(crystals_button.items):
                            crystals_button.state = 0
                            crystals_button.remove_all_items()
```

**Comment by CovenEsme** [comment]
Fixed an issue where, regardless of which button you click, 5 gratitude crystals would be added to the counter when using sphere tracking.

Also makes sure the crystal counter doesn't visually update until you select how many crystals you found. Previously, it would add 1 to the counter and *then* ask you how many you found

**Review by CovenEsme — APPROVED** [review]

**Pull request merged**